src/filemenu/filemenu_common.c

```diff
@@ -732,6 +891,11 @@ void filemenu_draw_cursor(MenuPanel* menu, s32 baseX, s32 baseY, s32 width, s32
     }
 }
 
+#if VERSION_PAL
+#define PAGE_4 (3)
```

Comment:
Should probably use the same enum as in `filemenu_main.c`.

Reply:
Yes